Add write key binding

ojung opened the pull request.

ojung committed:
Skip empty commands

ojung committed:
Use built-in cut command

ojung committed:
Only log when no config is found

ojung committed:
Report error on empty file

gesundkrank commented on shell/key-bindings.zsh:
```diff
@@ -10,4 +10,10 @@
   bindkey '^R^R' tome-show-widget
+
+  tome-write-command() {
+    selected=$(fc -lrn 0 | fzf | sed 's/\\n/\n/g')
```
I'd prefer using `zle` to fetch the selected command from the history. This way you get the raw command and don't have to replace any escaped newlines. 

see https://github.com/junegunn/fzf/blob/master/shell/key-bindings.zsh#L106

ojung committed:
Do not store empty commands

ojung committed:
Use more involved fzf invocation

inspired by https://github.com/junegunn/fzf/blob/master/shell/key-bindings.zsh#L110-L111

ojung committed:
Make write read from stdin

It reads until EOF

gesundkrank commented on src/cmd/write.go:
```diff
@@ -28,7 +27,7 @@ var writeCmd = &cobra.Command{
 		tome.Check(err)
 
-		fmt.Printf("Stored command: %s\n", command.String())
+		tome.LogInfof("Stored command: %s\n", command.String())
```
Can't we just keep the old version and add a proper logger that supports log levels, later?

ojung replied:
The log statement is not relevant for a user. The return code should be enough. So I would rather drop the log rather than sticking to the current behavior. wdyt?

gesundkrank replied:
fine by me

ojung committed:
Do not log on successful write
